Modules/Filtering/ImageStatistics/include/itkLabelOverlapMeasuresImageFilter.hxx

```diff
@@ -375,15 +375,20 @@ LabelOverlapMeasuresImageFilter<TLabelImage>::GetFalsePositiveError() const -> R
 {
   RealType numerator = 0.0;
   RealType denominator = 0.0;
+
+  auto sourceImg = const_cast<LabelImageType *>(this->GetInput(0));
+  auto nVox = sourceImg->GetBufferedRegion().GetNumberOfPixels(); // TP+FP+FN+TN
```

Comment:
Just wondering, why is the `const` casted away from the input (`const_cast<LabelImageType *>`)?

Reply:
I have not checked whether the return value is const or not. Plus, whether this const property should be away or not..

I refer the suggestion from @dzenanz and quickly made a suggestion.
https://github.com/InsightSoftwareConsortium/ITK/issues/3490#issuecomment-1194438181

Therefore, if you think keeping const is better and safer.. then I think we can edit them and test them, too.

Reply:
It might not compile without `const_cast`? Am I wrong?

Reply:
It does compile without it. I will force-push. Thanks for catching this Niels!